support-data-rest/src/main/java/org/uniprot/api/support/data/configure/service/UniParcConfigureService.java

```diff
@@ -54,6 +54,10 @@
                 .collect(Collectors.toList());
     }
 
+    public List<UniProtReturnField> getUniParcDatabaseResultFields() {
```

Comment:
should we change this path to crossReference instead of database?

Reply:
changed it to getUniParcEntryResultFields